content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -3,30 +3,31 @@ title: Impact of the CircleCI Security Incident on the Datadog Agent
 kind: faq
 ---
 
-On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures Datadog is taking to mitigate any risks to our customers.
+On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking of stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures Datadog is taking to mitigate any risks to our customers.
 
 <div class="alert alert-info">
-<strong>Note</strong>: As of January 12th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+<strong>Note</strong>: As of January 16th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
 </div>
 
 ## The affected key
 
-The RPM GPG signing key has fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
+The impacted RPM GPG signing key has the fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases, Agent 6 releases up to (and including) v6.13.0, DogStatsD 6 releases and DogStatsD 7 releases up to (and including) v7.23.1.
```

Comment:
What "DogStatsd releases" refer to is likely to be confusing to many customers. It's not a widely documented flavor of the agent (AFAIK we only document it in passing here: https://docs.datadoghq.com/agent/#overview). In any case, we should call it "standalone DogStatsD releases" to completely differentiate it from the Agent releases. I think we should also mention the package name for additional clarity (this might make the message a bit cluttered though). So I'd suggest the following:

```suggestion
The impacted RPM GPG signing key has the fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign:

* Agent 5 releases and Agent 6 releases up to (and including) v6.13.0 (`datadog-agent` package)
* Standalone DogStatsD 6 releases and Standalone DogStatsD 7 releases up to (and including) v7.23.1 (`datadog-dogstatsd` package)
```